### DISCUSSION TITLE
Tree Support boundaries not emmitted to Exclude Object?

### DISCUSSION THREAD

**Discussion by 5teveO** [OrcaSlicer/OrcaSlicer#1328] [Q&A]
Hi All.. 

I'll preface this by saying I am new to Orca and as such still learning. I am not new to 3d printing and generally use PS.. Orca has some great features and I am wanting to get along with it.

I had an issue last night,  Basic control box for my CNC, it has a few holes that need Tree supports just for a bit of help. I first encountered an issue as the linewidths were 0 - tree supports fail in that instance or did on this model (cant remember the error but thats incidental at the moment)

I'm running KAMP on klipper. II sliced the model - started the print in Fluidd.. and off it went. It was only then that I noticed that the KAMP Adaptive Purge line was located where i would expect it if no tree support was there, and then the tree support proceeded to print over the top of the purge line. The print still worked, so no major issues.. it just appeared that the tree supports addition to the model boundaries wasnt emiited in the polygon details for the exclude object.

I understand the skirt is ignored as that surround multiple objects, so wondering if the tree supports work in this way as they could be shared between objects?

Thank in advance

Steve

**Comment by grrminator** [comment]
I am also having this issue. Did you get it resolved?

**Comment by StdVectorBool** [comment]
This happens to me as well, the KAMP purge line ends up going through the support base:
![image](https://github.com/SoftFever/OrcaSlicer/assets/4926951/f4c2f6a8-4dfb-46e3-abe4-c2d2703ef49e)

**Comment by Lord-Pieter** [comment]
Same problem here. Waiting for a fix.
